api/src/Migrations/Version253.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version253 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Setting all users agree_terms_use to null so they are forced to view the new terms of service and agree.';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('UPDATE dd_user SET agree_terms_use = null');
+        $this->addSql('UPDATE dd_user SET agree_terms_use_date = null');
```

Comment:
If we're excluding lays from this exercise now then you will probably want to add a where clause for != `ROLE_LAY_DEPUTY` here.